src/styles/keymap.css

```diff
@@ -112,8 +112,12 @@ button {
   }
 
   /* don't print UI elements such as "Successfully printed" toast and the print button */
+  /* stylelint-disable selector-class-pattern */
```

Comment:
For some reason stylelint didn't accept a second `stylelint-disable-line` annotation